### PR TITLE
Reduce number of YAML files for ha_textmode_skip_reg*

### PR DESCRIPTION
**Pull request by rakoenig**
- Related ticket: https://progress.opensuse.org/issues/153049
- Needles:  - 
- [Verification runs](https://openqa.suse.de/tests/overview?build=rakoenig%2Fos-autoinst-distri-opensuse%23reduce_yaml_ha_textmode&version=15-SP6&distri=sle) 
- [Related MR on Gitlab](https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/71)

### CODE REVIEW

File: tests/yam/validate/validate_ha.pm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that the product installed by agama via /etc/os-release
```
Comment (jknphy) [inline tests/yam/validate/validate_ha.pm:6]: this only validate the repo not the /etc/os-release, I remember we already have some module to validate repos from other test suites.

File: tests/yam/validate/validate_ha.pm [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use Config::Tiny;
```
Comment (jknphy) [inline tests/yam/validate/validate_ha.pm:13]: check for unused libs.

File: tests/yam/validate/validate_ha.pm (hunk 1 above)
Comment (jknphy) [reply tests/yam/validate/validate_ha.pm:6]: additionally to validate that is minimal or not I think we have some other module to check patterns.

**Comment by jknphy** [comment]
Once you succeed having single yaml, please use folder to store both yaml (they are two different test cases in my understanding) in `schedule/yam/test_cases`.
(when there are still more than one we can keep it in existing folder to review later).

File: tests/yam/validate/validate_ha.pm (hunk 1 above)
Comment (lemon-suse) [reply tests/yam/validate/validate_ha.pm:6]: Maybe this tests/console/validate_repos.pm can be used instead.

Comment (lemon-suse) [reply tests/yam/validate/validate_ha.pm:6]: This tests/console/validate_packages_and_patterns.pm is for validate packages and patterns, need check whether can be used for our test.

Comment (rakoenig) [reply tests/yam/validate/validate_ha.pm]: Switched to `validate_repos` now as a validation module. This introduced a new test data file.

**Comment by jknphy** [comment]
> Once you succeed having single yaml, please use folder to store both yaml (they are two different test cases in my understanding) in `schedule/yam/test_cases`. (when there are still more than one we can keep it in existing folder to review later).

can be moved to test_cases/ folder, seems so, right?

**Comment by rakoenig** [comment]
> can be moved to test_cases/ folder, seems so, right?

I did the move, even if we have no VR for ppc64le (infrastructure poblems).

**Pull request merged**